js/jquery.validation.js

```diff
@@ -85,40 +80,83 @@
             
             // Handler function for submit event.            
             $form.on('submit.validate', function(event) {
-                var $form = $(this);
 
-                if ( typeof FormData === "function" ) {
-                    // AJAX file upload supported
+                // Default url used for validation service.
+                if ( typeof widget.options.url === 'undefined' ) {
+                    var url = $form.attr('action');
+                }
+
+                if ( typeof FormData === "function"
```

Comment:
@PeterChaplin There are 3 different ways here to construct the form data ... can you confirm you have tested each code path for handling method overloading with the "_method" form variable.